libraries/aws_network_acls.rb

```diff
@@ -0,0 +1,106 @@
+# frozen_string_literal: true
+
+require 'aws_backend'
+
+class AwsNetworkACLs < AwsResourceBase
```

Comment:
Can we give the name as aws_ec2_network_acls. Why I am saying is it will be easy to find ec2 and it comes into a single category.

Reply:
no it does not make sense to group that way .. in that case we need to group every resource in our existing packs also to that

Reply:
ok